lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,119 @@
+module ScrabbleSinatra
+  class ScrabbleSinatra
+    VALUES = {a: 1, e: 1, i: 1, o: 1, u: 1, l: 1, n: 1, r: 1, s: 1, t: 1,
+      d: 2, g: 2,
+      b: 3, c: 3, m: 3, p: 3,
+      f: 4, h: 4, v: 4, w: 4, y: 4,
+      k: 5,
+      j: 8, x: 8,
+      q: 10, z: 10
+    }
+
+    # this returns the score for each letter, total score for the word, and labels winning word
+    def self.scoring(word_array, score_method)
+      display_array = []
+      # checks if there is more than one word entered; used to label winning word
+      winning_word = highest_score_from(word_array, score_method) if word_array.length > 1
+      word_array.each do |word|
+        letter_hash = score_multiple(word.split(//))
```

Comment:
Since you're not using the result of `word.split` in this method, it would make sense to just pass in the `word` here and then split it into an array within the `score_multiple` method.